redfish_client/resource.py

```diff
@@ -42,6 +42,23 @@ def __init__(self, connector, oid=None, data=None):
             self._content = data
             self._headers = {}
 
+        if isinstance(self._content, dict):
+            for key in self._content.keys():
+                if key.startswith("#") and "target" in self._content[key]:
```

Comment:
I was expecting to simply loop `Actions` element. Here you loop all data and see if anything starts with a `#`. Is this part of Redfish standard? I may be missing something.

Reply:
Problem here is that Actions sometimes include more "levels" ("OEM" > "Lenovo") instead of just actions. In order to generate Lenovo methods user would have to nagivate to `systems.Actions.Oem.Lenovo`.

As @tadeboro suggested we could use: `system.Actions["#ComputerSystem.Reset"].execute(dict(my: "params", go: "here"))`

Whit this approach we wouldn't need to generate methods dynamically, if user would try to call `execute` on non `Action` we could just throw an error.